src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -131,9 +133,10 @@
 
 	private static final int REF = -1;
 	private static final int NO_REF = -2;
+	public static final int DEFAULT_SERIALIZER_SIZE = 68;

```

Comment:
Although extracting constants is generally a good idea, I do not think we should expose this publicly. Please inline the constant again.